tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/packaging/ModulePushTestCase.java

```diff
@@ -70,13 +70,20 @@ public void testPush() throws Exception {
     
         Assert.assertTrue(Files.exists(projectPath));
         Assert.assertTrue(Files.isDirectory(projectPath));
+
+        Path modulePath = projectPath.resolve("src").resolve(moduleName);
     
         // Create module
         balClient.runMain("add", new String[]{moduleName}, envVariables, new String[]{}, new LogLeecher[]{},
                 projectPath.toString());
     
-        Assert.assertTrue(Files.exists(projectPath.resolve("src").resolve(moduleName)));
-        Assert.assertTrue(Files.isDirectory(projectPath.resolve("src").resolve(moduleName)));
+        Assert.assertTrue(Files.exists(modulePath));
```

Comment:
We can get rid of asserting whether file exists because Files.isDirectory(modulePath) will fail anyway if the file doesn't exist. WDYT?
Let's correct this in other similar places too.